### DISCUSSION TITLE
heavy stable-isotope labels

### DISCUSSION THREAD

**Discussion by quarksome** [OpenMS/OpenMS#5843] [Q&A] [answered]
Hi all

I'm wondering if pyOpenMS can parse 13C and 15N in chemical formula. I tried using '[13C]' but it did not recognize it as 13C isotope. I am curious because I want to add stable-isotope labeled internal standard in the AccurateMassSearchEngine() database.

Thanks!

**Comment by jpfeuffer** [comment, marked as answer]
Hi!

Yes, this should be possible with "(13)C".
You can also mix isotopes in the same formula "(13)C2(12)C3".

See https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/release/latest/html/classOpenMS_1_1EmpiricalFormula.html#details

**Comment by Stef-RNA** [comment]
Hi,
does this also work for NASE and RNA? in which file would I need to make the change?
Thanks!
Stef
